src/styles/styles.js

```diff
@@ -361,6 +361,18 @@ const styles = {
         backgroundColor: themeColors.offline,
     },
 
+    floatingActionButton: {
+        backgroundColor: themeColors.buttonSuccessBG,
+        position: 'absolute',
+        height: variables.componentSizeNormal,
+        width: variables.componentSizeNormal,
+        right: 20,
+        bottom: 24,
+        borderRadius: 999,
+        alignItems: 'center',
+        justifyContent: 'center',
```

Comment:
Small thing, but there is an issue with `react-native-web` and we are not seeing a pointer when using a `Pressable`. But also seems like this issue has been fixed recently.

https://github.com/necolas/react-native-web/issues/1764

Reply:
I finally got the app to run on the browser and the cursor really does not change to pointer.

I added `cursor: pointer` to the floatingActionButton style, and it does work. But I noticed the design does not fit very well on the browser.

The image shows it below. I am leaving for a couple of hours but will take a look at this when I come back. But I am not sure how to style differently for each platform. Do you have any suggestion on that?

![problem](https://i.ibb.co/W0gHFc2/problem.png)

Reply:
To clarify, this issue can be resolved by updating `react-native-web` version to latest in `package.json`?

```
    "react-native-web": "^0.14.1",
```

Reply:
We should be able to remove `cursor: 'pointer'` then. 
No need to worry about the design as other components will be relocated soon.

Reply:
You are correct! I tested updating the `react-native-web` as you mentioned, removed the `cursor: pointer` from the style and it worked just fine!
```
"react-native-web": "^0.14.1",
```

Reply:
This is something I cannot do, right? Update the package.json and package-lock.json.

I tried but got lint error, I guess I will just undo it. But it is nice to check that it works.

Reply:
Hmm, I think you should be able to update. Which linter error did you get?

Reply:
First I just really tried to update the package and add the `pacakge.json` file to the commit. It complained that `package-lock.json` must be in sync.

Then I ran `npm install` and added both the new `package.json`, which had a new package on the dev dependencies now besides the change on the `react-native-web`. It was the `mime-types` package I believe.

When I pushed these new changes it gave me a big error. Sorry, I don't remember it very well. I read it and could not find anything that helped me, that's why I just assumed I was not supposed to do it.

I can try again and give you a better error description.

Reply:
Guess I can just check the history, right? Haha

Here is the error log:

```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/Expensify/react-native-onyx.git
npm ERR! 
npm ERR! Warning: Permanently added the RSA host key for IP address '140.82.113.4' to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-01-14T04_38_59_667Z-debug.log
Error: Process completed with exit code 1.
```

Reply:
Ah no problem I'll create a fix for this. I think the problem with you ssh keys perhaps 🤔